### PR TITLE
Regenerated test languages

### PR DESCRIPTION
**Pull request by kthoms**
Signed-off-by: Karsten Thoms <karsten.thoms@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.ide.tests/model/generated/IndentationAwareUiTestLanguage.ecore [hunk 1]
```diff
@@ -9,12 +9,12 @@
         eType="#//OtherTreeNode" containment="true"/>
   </eClassifiers>
   <eClassifiers xsi:type="ecore:EClass" name="TreeNode">
-    <eStructuralFeatures xsi:type="ecore:EAttribute" name="name" eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EString"/>
+    <eStructuralFeatures xsi:type="ecore:EAttribute" name="name" eType="ecore:EDataType platform:/resource/org.eclipse.emf.ecore/model/Ecore.ecore#//EString"/>
     <eStructuralFeatures xsi:type="ecore:EReference" name="children" upperBound="-1"
         eType="#//TreeNode" containment="true"/>
   </eClassifiers>
   <eClassifiers xsi:type="ecore:EClass" name="OtherTreeNode">
-    <eStructuralFeatures xsi:type="ecore:EAttribute" name="name" eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EString"/>
```
Comment (cdietrich) [inline org.eclipse.xtext.ide.tests/model/generated/IndentationAwareUiTestLanguage.ecore]: am not sure if this change is good. cc @szarnekow

Comment (cdietrich) [reply org.eclipse.xtext.ide.tests/model/generated/IndentationAwareUiTestLanguage.ecore]: @kthoms @szarnekow any ideas?

Comment (szarnekow) [reply org.eclipse.xtext.ide.tests/model/generated/IndentationAwareUiTestLanguage.ecore]: I think it's ok here - the test lang was probably not regenerated as part of the entire test lang generator before.

**Comment by cdietrich** [comment]
@kthoms builds are red http://services.typefox.io/open-source/jenkins/job/xtext-core/job/kth_regen_testlang/2/

**Comment by kthoms** [comment]
```Missing requirement: org.eclipse.xtext.testlanguages 2.13.0.v20171002-1320 requires 'bundle org.eclipse.xtext.xbase.testing 0.0.0' but it could not be found```
I guess this since it is a testlang in core w/o access to xbase.

I will manually remove the dependency from there, but we might have to adjust the generator to avoid this when Xbase isn't present. Another issue.

**Comment by cdietrich** [comment]
so simply rebase and regenerate @kthoms

**Pull request merged**